web/app/models/build.ts

```diff
@@ -0,0 +1,59 @@
+import {BuildStatus, FastlaneStatus, fastlaneStatusToEnum} from '../common/constants';
+
+const SHORT_SHA_LENGTH = 6;
+
+export interface BuildResponse {
+  number: number;
+  project_id: string;
+  status: FastlaneStatus;
+  duration: number;
+  description: string;
+  trigger: string;
+  lane: string;
+  platform: string;
+  parameters: {[param: string]: string};
+  build_tools: {[tool: string]: string};
+  clone_url: string;
+  branch: string;
+  ref: string;
+  sha: string;
+  timestamp: string;
+}
+
+export class Build {
+  readonly number: number;
+  readonly projectId: string;
+  readonly status: FastlaneStatus;
+  readonly duration: number;
+  readonly description: string;
+  readonly trigger: string;
+  readonly lane: string;
+  readonly platform: string;
+  readonly parameters: {[param: string]: string};
+  readonly buildTools: {[tool: string]: string};
+  readonly cloneUrl: string;
+  readonly branch: string;
+  readonly ref: string;
+  readonly sha: string;
+  readonly shortSha: string;
+  readonly date: Date;
+
+  constructor(buildSummary: BuildResponse) {
+    this.number = buildSummary.number;
+    this.projectId = buildSummary.project_id;
+    this.description = buildSummary.description;
+    this.trigger = buildSummary.trigger;
+    this.lane = buildSummary.lane;
+    this.platform = buildSummary.platform;
+    this.parameters = buildSummary.parameters;
```

Comment:
Looks like some changes overlap with https://github.com/fastlane/ci/pull/936, something went wrong with the branches?

Reply:
yeah this should be targeting that branch instead of master. I changed it but doesn't look like it went through.

Reply:
👍